packages/react-shortcuts/src/ShortcutManager/ShortcutManager.tsx

```diff
@@ -60,27 +61,36 @@ export default class ShortcutManager {
     }
   };
 
-  private updateMatchingShortcuts() {
+  private updateMatchingShortcuts(event: KeyboardEvent) {
     const shortcuts =
       this.shortcutsMatched.length > 0 ? this.shortcutsMatched : this.shortcuts;
 
-    this.shortcutsMatched = shortcuts.filter(({keys, node, ignoreInput}) => {
-      if (isFocusedInput() && !ignoreInput) {
-        return false;
-      }
-
-      const partiallyMatching = arraysMatch(
-        this.keysPressed,
-        keys.slice(0, this.keysPressed.length),
-      );
+    this.shortcutsMatched = shortcuts.filter(
+      ({keys, modifierKeys, node, ignoreInput}) => {
+        if (isFocusedInput() && !ignoreInput) {
+          return false;
+        }
+
+        if (
+          modifierKeys &&
+          !modifierKeys.every(key => event.getModifierState(key))
+        ) {
+          return false;
+        }
+
+        const partiallyMatching = arraysMatch(
+          this.keysPressed,
+          keys.slice(0, this.keysPressed.length),
+        );
 
-      if (node) {
-        const onFocusedNode = document.activeElement === node;
-        return partiallyMatching && onFocusedNode;
-      }
+        if (node) {
```

Comment:
Weird spacing

Reply:
What is the expected spacing?

Reply:
I mean in github it's showing it as like a different character or something. Probably running prettier on it will fix?